src/main/permissionsManager.ts

```diff
@@ -105,13 +105,17 @@ export class PermissionsManager extends JsonFileManager<Permissions> {
             return false;
         }
 
+        // Exception for embedded videos such as YouTube
+        // We still want to ask permission to do this though
+        const isExternalFullscreen = permission === 'fullscreen' && parsedURL.origin !== serverURL.origin;
+
         // is the requesting url trusted?
-        if (!(isTrustedURL(parsedURL, serverURL) || (permission === 'media' && parsedURL.origin === serverURL.origin))) {
+        if (!(isTrustedURL(parsedURL, serverURL) || (permission === 'media' && parsedURL.origin === serverURL.origin) || isExternalFullscreen)) {
```

Comment:
Do we want to ask permission or not for external full screen? Is this supposed to be `!isExternalFullscreen`?

Reply:
We do, there's an additional `!` at the beginning of that statement.

Reply:
Ah gotcha, didn't see that it was added in the two parenthesis there